### DISCUSSION TITLE
SimpleShell not run after update to dotnet 8

### DISCUSSION THREAD

**Discussion by 19bartek92** [RadekVyM/SimpleToolkit#42] [Q&A] [answered]
Hi, 

after adding SimpleShell to the project with UseSimpleShell I get
 `Unable to resolve service for type 'Microsoft.Maui.IPropertyMapper' while attempting to activate 'SimpleToolkit.SimpleShell.Handlers.SimpleShellHandler'.`
 
 It appears as well on a new project as on the updated to dotnet 8, "HamburgerMenuApp" sample. 
 Am I missing something?
 
 **SimpleShell 4.1.0
 Core 4.0.0**

**Comment by RadekVyM** [comment, marked as answer]
Hi @19bartek92,

is it happening on macOS? If so, please try to update your workloads/VS for Mac. I have dealt with this issue too. I have upgraded my VS for Mac today and it finally works.

If that does not help, you can try to use SimpleShell 4.0.0. I have no idea where the problem is/was. I created the SimpleShell 4.1.0 package the same way I had created all the previous versions.

**Reply by 19bartek92** [reply]
Yes, I am on mac with rider. After updating the workloads everything seems to be fine. 
Thanks for your help 😉

**Comment by Hoosje1958** [comment]
Is it possible to have a submenu in SimpleToolkit? I was trying to put the ISVisible in the TAB item, but that doesn't work?
